### PR TITLE
fixed issue

### PR DESCRIPTION
**Pull request by rajshukla1102**
Close https://github.com/unifyai/ivy/issues/10421

### CODE REVIEW

**Comment by rajshukla1102** [comment]
@hirwa-nshuti look into once. Thankyou!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Testing Function
 # global
+import numpy as np
```
Comment (Dsantra92) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py:3]: Is is required here?

**Review by Dsantra92 — CHANGES_REQUESTED** [review]
Numpy converts each value to int64. You might want to cast each value in the tuple to int64. It should fix the current tests.

**Review by rajshukla1102 — COMMENTED** [review]
@Dsantra92 I have made the changes. please look into it once.

**Comment by rajshukla1102** [comment]
@zhumakhan Thanks for the review but currently I think ivy is not supporting orders. It is the reason I didn't add support. Thank you!

**Comment by zhumakhan** [comment]
Yeah, you are right.
Lets merge then!

**Pull request merged**